caikit/runtime/servicers/global_predict_servicer.py

```diff
@@ -137,10 +137,10 @@ def Predict(self, request, context):
             response (object):
                 A Caikit Library data model response object
         """
-        desc_name = request.DESCRIPTOR.name
-        outer_scope_name = "GlobalPredictServicer.Predict:%s" % desc_name
+        request_name = request.DESCRIPTOR.name
+        outer_scope_name = "GlobalPredictServicer.Predict:%s" % request_name
```

Comment:
Funny that for these scopes we're using `%` encoding! Nothing to change (and I have plans to move these lines around a bit, so no need to mess with them), but some day we should actually get explicit about when to use `f"{}"`, `"{}".format(...)`, and `"%s" % ...` style string interpolation (and of course lazy log interpolation with `"%s", ...`!)